app/models/extra_mobile_data_request.rb

```diff
@@ -38,7 +37,7 @@ def self.problem_statuses
   end
 
   def self.statuses_available_to_mnos
-    statuses.keys - %w[cancelled unavailable requested]
+    statuses.keys - %w[cancelled unavailable]
```

Comment:
Are MNO's allowed to use 'new' ?

Reply:
You're right, they shouldn't, according to the designs. I'm gonna do a follow-up PR which will update the design for MNOs, I'll fix this as part of that PR.